grpc-proto/src/test/java/ValuesTest.java

```diff
@@ -0,0 +1,323 @@
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import io.stargate.grpc.Values;
+import java.math.BigDecimal;
+import java.math.BigInteger;
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+import java.nio.ByteBuffer;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.UUID;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+public class ValuesTest {
+
+  @Nested
```

Comment:
Quick question on this: I have seen use of `@Nested`, but was curious as to its benefits -- is it just for improved grouping of test results, or such? Nothing against the use just want to understand benefits over either including tests directly in main test class, or as separate main-level test cases.